README.md

```diff
@@ -141,11 +142,14 @@ The format of the _installer-config.txt_ file and the current defaults:
                               # RPi board, your network device might be named differently. This will result in the
                               # board having no network connectivity.
     ifname=eth0
-    ip_addr=dhcp
-    ip_netmask=0.0.0.0
-    ip_broadcast=0.0.0.0
-    ip_gateway=0.0.0.0
-    ip_nameservers=
+    ip4_addr=dhcp # options are 'disable', 'dhcp', or an IPv4 address
+    ip4_prefixlength=0
```

Comment:
Forgive my ignorance, but why did you change `netmask` to `prefixlength`?
I thought `netmask` is a better known term.